src/py.rs

```diff
@@ -0,0 +1,305 @@
+use std::cmp::Ordering;
```

Comment:
[In reply to Ivan Rybin]
A general comment the naming of the module py is confusing, maybe it should be dict? Also would be good to add some docs to both structures and module description.

Reply:
Especially because the module is public it is better to have some docs.